frontend/src/pages/GoodBye/GoodBye.spec.tsx

```diff
@@ -22,16 +22,23 @@ describe('GoodBye', () => {
   });
 
   it('should render', () => {
-    render(<GoodBye />, { wrapper: BrowserRouter });
```

Comment:
Note: `wrapper: BrowserRouter` is no longer supported. the change is fairly simple though, just had to directly wrap the `Goodbye` component with `BrowserRouter`